corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -225,9 +238,12 @@ public int loadTable(String namespace, String tablename, int numItems, int batch
                     TableName.getDefaultInstance().getClass(),
                     optionsBuilder.build());
 
-            TableName dummyVal = TableName.newBuilder().setNamespace(namespace).setTableName(tablename).build();
-            log.info("Loading {} items in {} batchSized transactions into {}${}",
-                    numItems, batchSize, namespace, tablename);
+            byte[] array = new byte[itemSize];
+            new Random().nextBytes(array);
```

Comment:
why we need this?

Reply:
if we don't throw in random bytes, the compression reduces the actual payload size making it hard to test things like disk full

Reply:
how does adding random bytes solve it?  can we add a comment about this?

Reply:
adding random bytes prevents compression from reducing the size of the written value. random bytes don't compress well.

Reply:
comments added.